pkg/spec/cluster.go

```diff
@@ -70,6 +71,10 @@ type ClusterSpec struct {
 	// The vaild range of the size is from 1 to 7.
 	Size int `json:"size"`
 
+	// Image is the expected etcd image name
+	// for user Get image from private registry
+	// default is quay.io/coreos/etcd
+	Image string `json:"image"`
```

Comment:
BaseImage? We might attach version tags, for example quay.io/coreos/etcd:v3.0.17